GOVERNANCE.md

```diff
@@ -0,0 +1,179 @@
+# Prometheus Governance 2.0
+
+This document describes the rules and governance of the project. It is meant to be followed by all the developers of the project and the Prometheus community. This governance is an open, living document, and will continue to evolve as the community and project change.
+
+It outlines the roles and responsibilities of the Steering Committee, the decision-making process, and the election procedures for committee members. It also details the contributor ladder, including the roles of Contributor, Member, and Maintainer, and the requirements and privileges associated with each role.
+
+The Prometheus Steering Committee is the governing body of the Prometheus project, providing decision-making and oversight pertaining to the project bylaws, sub-organizations[^1], and financial planning. The Steering Committee also defines the project values and structure.
+
+## Values
+
+The Prometheus developers and community are expected to follow the values defined in the [CNCF charter](https://www.cncf.io/about/charter/), including the [CNCF Code of Conduct](https://github.com/cncf/foundation/blob/master/code-of-conduct.md). Furthermore, the Prometheus community strives for kindness, giving feedback effectively, and building a welcoming environment. The Prometheus developers default to deciding by consensus and only resort to conflict resolution by a majority vote if consensus cannot be reached.
+
+## Roles
+
+We have a contributor ladder that includes the roles of Contributor, Member, Maintainer and Steering Committee. The steering committee responsibilities and elections are outlined below. The rest of the roles are detailed in [ROLES.md](#roles.md).
```

Comment:
Note that `#roles.md` will not point to the file `roles.md`. I guess you meant to not use `#` here.